nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/admin.js

```diff
@@ -3,6 +3,10 @@
 module.exports = function () {
   this.Then('I can see the administration menu', () => this.ui.drawer.administration.isVisible().should.be.true);
 
+  this.Then('I cannot see the administration button', () => this.ui.adminButton.isVisible().should.be.false);
+
+  this.Then('I cannot see the administration menu', () => this.ui.drawer.administration.isVisible().should.be.false);
```

Comment:
One is the button in the side menu and the other is the menu that is displayed when I click on that button, so in a way, yes, although the elements are different. I did it this way as the ticket (NXP-21730) specifically mentioned the Admin Center but I've now removed the second line.